libs/portals/admin/form-system/src/hooks/previewControlReducer.ts

```diff
@@ -0,0 +1,123 @@
+import { FormSystemStep, FormSystemGroup } from '@island.is/api/schema'
```

Comment:
I feel like this would belong in a folder named reducers or something else, not in the hooks folder